### PR TITLE
[bridge-indexer] some logs and todos

### PR DESCRIPTION
**Pull request by longbowlu**
## Description 

as title

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

**Review by dariorussi — COMMENTED** [review]
This looks fine to me. I'll let others comment as needed and I'll come back and accept if we do not get an approval later

File: crates/sui-bridge-indexer/src/main.rs
Comment (Bridgerz) [inline crates/sui-bridge-indexer/src/main.rs:160]: why not use join?

File: crates/sui-indexer-builder/src/indexer_builder.rs
Comment (Bridgerz) [inline crates/sui-indexer-builder/src/indexer_builder.rs:170]: Isn't this the live task?

File: crates/sui-bridge-indexer/src/main.rs
Comment (longbowlu) [reply crates/sui-bridge-indexer/src/main.rs:160]: * `join_all` won't return until all tasks are finished. So if one task errors, we won't immediately know
* `try_join!` gives a vector of `Result<>` which i don't bother to unwrap

this way feels the easiest and less error-prone

File: crates/sui-indexer-builder/src/indexer_builder.rs
Comment (longbowlu) [reply crates/sui-indexer-builder/src/indexer_builder.rs:170]: `let latest_backfill_task = backfill_tasks.first();` gives you the backfill task with the highest target checkpoint.

This has nothing to do with live task. Are you asking why we are using a variable called `latest_backfill_task` in a if statement that seems to be related to live task?

File: crates/sui-bridge-indexer/src/main.rs
Comment (longbowlu) [reply crates/sui-bridge-indexer/src/main.rs:160]: > join_all won't return until all tasks are finished. So if one task errors, we won't immediately know


if you rmr it took us a while to figure out why there is no Eth backfill tasks, this is why

File: crates/sui-indexer-builder/src/indexer_builder.rs
Comment (Bridgerz) [reply crates/sui-indexer-builder/src/indexer_builder.rs:170]: never mind. I was thinking the variable name was misnamed but seems appropriate. We assume the first backfill_task is the live task... makes sense.

**Review by Bridgerz — APPROVED** [review]

**Pull request merged**
